languages/x86-64-assembly/maintainers.md

```diff
@@ -4,8 +4,4 @@ These awesome people help maintain the x86-64 Assembly track.
 
 ## Senior Maintainers
 
-TODO: add senior maintainers
-
-## Contributing Maintainers
-
-TODO: add contributing maintainers
+- Johan Berg (Slack: `@Johan Berg`, GitHub: [bergjohan](https://github.com/bergjohan))
```

Comment:
Why remore the `## Contributing Maintainers`. Will there be no new ones?

Reply:
I presume because the section is empty.

So if contributing maintainers appear, the section can be added back?

Reply:
Right now, I am the only maintainer for this track. If someone shows up in the future, I'll bring back the section :)

Reply:
Just leaves room for errors if a Contributing Maintainer wants to put his name. If there are none I'd still do it like this: https://github.com/exercism/v3/blob/master/languages/csharp/maintainers.md

Reply:
Just my opinion :)

Reply:
@tehsphinx Yeah, that's a good idea! I created another PR for this.